integration-tests/src/test/java/oracle/weblogic/kubernetes/TestConstants.java

```diff
@@ -223,12 +223,15 @@ public interface TestConstants {
   public static final String WEBLOGIC_INDEX_KEY = "wls";
   public static final String KIBANA_INDEX_KEY = "kibana";
   public static final String KIBANA_NAME = "kibana";
-  public static final String KIBANA_IMAGE = KIBANA_NAME + ":" + ELK_STACK_VERSION;
+  public static final String KIBANA_IMAGE_NAME = "phx.ocir.io/weblogick8s/test-images/docker/kibana";
```

Comment:
@ddsharpe @robertpatrick, what do you think? This seems like it violates the spirit of not having internal addresses in GitHub even though this image location is outside the corporate network.

Reply:
I'm leaning toward saying that the prefix should be coming from a Jenkins secret and not be in the source base.

Reply:
Modified code to take registry prefix from TEST_IMAGES_REPO variable